lib/src/span.dart

```diff
@@ -48,10 +48,11 @@ abstract class SourceSpan implements Comparable<SourceSpan> {
   /// Formats [message] in a human-friendly way associated with this span.
   ///
   /// [color] may either be a [String], a [bool], or `null`. If it's a string,
-  /// it indicates an ANSII terminal color escape that should be used to
-  /// highlight the span's text. If it's `true`, it indicates that the text
-  /// should be highlighted using the default color. If it's `false` or `null`,
-  /// it indicates that the text shouldn't be highlighted.
+  /// it indicates an ANSI terminal color escape that should be used to
```

Comment:
It's probably a good idea to link "ANSI terminal color escape" to https://en.wikipedia.org/wiki/ANSI_escape_code#Colors.

Reply:
Done.